packages/x-data-grid/src/hooks/features/columns/gridColumnsInterfaces.ts

```diff
@@ -6,6 +6,13 @@ export enum GridPinnedColumnPosition {
   RIGHT = 'right',
 }
 
+export enum PinnedPosition {
+  NONE,
+  LEFT,
+  RIGHT,
+  VIRTUAL,
+}
+
```

Comment:
There is a strong convention to prefix all the public exports with `Grid...`, e.g. `GridPinnedPosition`. If it's for internal purposes only, it should be exported from `@mui/x-data-grid/internals` instead, then it would be ok to skip the prefix.

Also this was originally local to `GridCell` so it was somewhat clear that it was for pinned columns. If it's exported globally it probably needs a "column" in there because "pinned position" can also refer to rows.

Reply:
Actually didn't mean to to export it publicly, just ended up with circular dependencies when having it in GridCell– didn't really understand the implications. Will switch it to internals rather.

Reply:
Moved it to internals and renamed.